javascript.md

```diff
@@ -0,0 +1,18 @@
+## Draft JavaScript Guidelines
+
+### General
+*  Use [Standard JS](https://standardjs.com/) to lint your code. A conistent approach to code layout, spacing and formatting to code makes it easier to switch between projects. By [adding it as a dev dependency](https://standardjs.com/index.html#install) it can be easily used in the terminal, [Webstorm](https://blog.jetbrains.com/webstorm/2017/04/using-javascript-standard-style/), and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).
```

Comment:
This should be a standard (pardon the pun) rather than a guideline.

Reply:
I agree